### PR TITLE
Add tv id as parameter

### PR DESCRIPTION
**Pull request by Szewcson**
Rs232 use tv id, this tv id may vary. On my TV I have found bug that blocks me from powering on tv when id is changed to other then initial one. In that case only using broadcast id can power on tv. When I changed tv id back to initial I'm able to power one TV using that id. So adding tv id as parameter seems beneficial.

### CODE REVIEW

**Comment by Szewcson** [comment]
I think that maybe also I should add id check in response parser.

File: src/lgtv_rs232/lgtv.py [hunk 1]
```diff
@@ -179,12 +179,16 @@ class LgTV():
...
-    def __init__(self, path, update=True):
-        # path is a string set to the location of the serial device, 
-        # e.g. "/dev/ttyUSB0"
+    def __init__(self, path, tv_id=0, update=True):
+        # path is a string set to the location of the serial device, e.g. "/dev/ttyUSB0"
+        # id is the tv id whcich can be set in tv options, as defalut value brodcast id is used
         self._volume = None
         self._muted = None
         self._input = None
+        if 0 <= tv_id <= 100:
```
Comment (zcline91) [inline src/lgtv_rs232/lgtv.py]: Should this be `0 < tv_id < 100`? 100 is not a valid id, right?

**Comment by zcline91** [comment]
Looks good. By adding id check in the response parser, do mean setting the _id attribute based on the response whenever the request() method is called?

File: src/lgtv_rs232/lgtv.py (hunk 1 above)
Comment (Szewcson) [reply src/lgtv_rs232/lgtv.py]: Right

**Comment by Szewcson** [comment]
> Looks good. By adding id check in the response parser, do mean setting the _id attribute based on the response whenever the request() method is called?

I just thinking about checking if id is equal returned id but setting appropriate id if broadcast was used is maybe also good idea. I think I will have to update parser in other [PR](https://github.com/zcline91/lgtv-rs232-zcline91/pull/2) so maybe keep it simple here, and not adding parser yet.

**Review by zcline91 — COMMENTED** [review]
Looks good. I just merged it.

**Pull request merged**